crypto/rand/drbg_ctr.c

```diff
@@ -84,12 +85,16 @@ __owur static int ctr_BCC_block(RAND_DRBG_CTR *ctr, unsigned char *out,
  */
 __owur static int ctr_BCC_blocks(RAND_DRBG_CTR *ctr, const unsigned char *in)
 {
-    if (!ctr_BCC_block(ctr, ctr->KX, in)
-        || !ctr_BCC_block(ctr, ctr->KX + 16, in))
-        return 0;
-    if (ctr->keylen != 16 && !ctr_BCC_block(ctr, ctr->KX + 32, in))
-        return 0;
-    return 1;
+    uint8_t in_tmp[48];
+    uint8_t num_of_blk = 2;
```

Comment:
I would refrain from using uint8_t here. The ctr_BCC_block parameters are unsigned char * and int so I would use them.